README.md

```diff
@@ -1,11 +1,11 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-
+# Capstone Project
 
 ## Built With
 
 - HTML
-- CSS
+- BOOTSTRATP
 - JAVASCRIPT
 
```

Comment:
- [ ] Please make use of GitHub Pages to deploy the web pages. The project must be deployed properly, and all pages, links, and images work as expected.
- [ ] The link to the deployed project must be present in the README.